### DISCUSSION TITLE
How to use with pi pico

### DISCUSSION THREAD

**Discussion by Rupeshshelar** [sandeepmistry/arduino-LoRa#492] [Q&A]
How to use it with pico and sx1278 for peer to peer communication

**Comment by matini250** [comment]
Use this library:
https://github.com/akshayabali/pico-lora

**Comment by morganrallen** [comment]
I do not have an picos to test on but I see no reason this should not work as it's an arduino compatible board. Try asking a specific question, or describe issues you are facing.
